### DISCUSSION TITLE
Needs approval problem for push user(Me)

### DISCUSSION THREAD

**Discussion by OmegaBeam69** [ROM-builders/temporary#17683] [General]
![Screenshot_20230131-161409_Brave](https://user-images.githubusercontent.com/62275326/215732346-1719d7f1-d0aa-40bd-860f-b14c55a7d406.png)

**Comment by Apon77** [comment]
I dont know solution of this

**Reply by OmegaBeam69** [reply]
I guess it’s a problem for new push user's? Like I'm push user for past 7 days
